tests/layer_tests/tensorflow_tests/test_tf_BitwiseShift.py

```diff
@@ -68,8 +68,8 @@ def create_bitwise_net(self, x_shape, y_shape, is_y_const, input_type, op_type):
     @pytest.mark.nightly
     def test_bitwise(self, x_shape, y_shape, is_y_const, input_type, op_type, ie_device, precision, ir_version,
                      temp_dir, use_legacy_frontend):
-        if ie_device == 'GPU':
-            pytest.skip("149424: Bitwise ops are not supported on GPU")
+        if ie_device == 'GPU' and input_type in [np.uint64]:
+            pytest.skip("149424: uint64 type is not supported on GPU")
```

Comment:
@pkowalc1, please create separate ticket for uint64 support on GPU and list this ticket number here

Reply:
@rkazants if you think there is a real need for uint64 support on GPU, please create a ticket and motivate it properly(e.g. maybe there is a business need). Currently, uint64 on GPU will be converted to int32 (info from V. Paramuzow), so there is a support but it it will not work properly in this case.

Reply:
The motivation is to be fully aligned with framework cases and capabilities. Our intention is to support not only known models but all other models about which we don't know.
Please create ticket and list here if you are responsible for resolving this task and if you want to change TF FE test sources. Otherwise, I can switch on these tests on GPU in a separate PR and create ticket on my own.

Reply:
@rkazants This PR is ready to be merged - all types except u64(as discussed above) should be working fine. However, if you want to create your own PR - no problem, please just close this one.